packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -239,10 +239,14 @@ export const MultichannelText = props => {
     )
 
     if (postbackButtonElements.length === 0) {
+      // Use texts[0] instead of textElements[0] as a body to avoid apply the markdownMeta function twice.
+      // If the markdownMeta function is applied twice, bold is replaced by italics.
+      const body = texts[0]
```

Comment:
seeing that texts comes from getText() function previously, would it be safer to do something like `texts[0] ?? ''` ?